src/userDashboard/ConstantsUserDashboard.js

```diff
@@ -80,6 +80,14 @@ const DATE_FORMAT = "yyyy-MM-dd";
 
 const DATE_FORMAT_FOR_DATEPICKER = "dd/MM/yyyy";
 
+const MAX_VALUE_BAR_GRAPH = {
+  WEEK: 15,

```

Comment:
is it on purpose that the min_y_axis_for_bar_graph is the same for week and for month?

Reply:
Yes. Because it's the same values (time/day) only for a longer period. If I change the value for month to be bigger, it looks like the time spent daily is getting shorter.